app/src/main/kotlin/org/mifos/mobile/cn/ui/mifos/beneficiaries/BeneficiariesActivity.kt

```diff
@@ -0,0 +1,66 @@
+package org.mifos.mobile.cn.ui.mifos.beneficiaries
+
+import android.annotation.SuppressLint
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.Button
+import android.widget.LinearLayout
+import androidx.core.content.ContextCompat
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import org.mifos.mobile.cn.R
+import org.mifos.mobile.cn.data.models.beneficiaries.ListBeneficiaries
+import org.mifos.mobile.cn.ui.adapter.BeneficiariesAdapter
+
+class BeneficiariesActivity : AppCompatActivity() {
+
+    @SuppressLint("ResourceAsColor")
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_beneficiaries)
+
+        val layoutListBeneficiaries = findViewById<LinearLayout>(R.id.mBtnListFocused)
```

Comment:
You don't have to initialize it here. You can directly use it.

Reply:
thanks for your comment :D